src/main/java/org/folio/rest/support/HridManager.java

```diff
@@ -246,6 +255,68 @@ private Future<String> getNextHrid(final HridSettings hridSettings, InventoryTyp
         Objects.toString(hridPrefix, ""), sequence.getLong(0)));
   }
 
+  /**
+   * Optimized version of getNextHrid that makes a single call to retrieve
+   * hridsettings and the next sequence value for the specified inventory type.
+   */
+  private Future<String> getNextHrid(InventoryType type) {
```

Comment:
Is the old getNextHrid method still used now this one has been introduced?

Reply:
the old getNextHrid method is not used anymore. I am very open to taking them out.

Reply:
Yes, please remove a now unused method.

Reply:
methods removed